enhancements/hide_container_mountpoints.md

```diff
@@ -0,0 +1,139 @@
+---
+title: hide-container-mountpoints
+authors:
+  - "@lack"
+reviewers:
+  - TBD
+approvers:
+  - TBD
```

Comment:
you can add myself, @mrunalp  and @umohnani8 here (to cover reviews for both cri-o and MCO bits)

Reply:
There's still only reviewers listed here, and no approvers

@haircommander @mrunalp @umohnani8 maybe one of you could agree to take point as final approver, and give your sense of where things stand so far? Thanks

Reply:
I have now added @haircommander @mrunalp and @umohnani8 as approvers, happy to add/remove any of them as required :)